include/clap/ext/draft/track-info.h

```diff
@@ -4,24 +4,76 @@
 #include "../../color.h"
 #include "../../string-sizes.h"
 
+// This extensions let the plugin query info about the track and device chain it's in.
+// It is useful when the plugin is created, to initialize some parameters (mix, dry, wet)
+// and pick a suitable configuartion regarding audio port type and channel count.
+//
+// The terminology used in this extension:
+// A track is a channel.
+// A channel contains one device chain and usually a mixer at the end.
```

Comment:
Maybe it's a idea to use the word 'track' here instead of channel? Especially since channel also refers to audio channels.

Reply:
+1 on reserving "channel" for referring only to a channel of audio in an audio stream for cases of multichannel processing

Reply:
How would you call the return channel inside a drum machine?

Reply:
`is_device_chain_return` would be consistent with the rest of the names.